### PR TITLE
Round missing corners - TextField + handled warnings

### PR DESCRIPTION
**Pull request by jankrzempek**
https://trello.com/c/RKDIeMEn/639-missing-rounding

### CODE REVIEW

**Review by SzymonGorowski — COMMENTED** [review]
If you used in three places the same piece of code. Maybe a good option will be to use `ViewModifier`? Anyway, good job! 👍

File: AirCasting/SessionsSynchronization/Adapters/Services/SessionUploadService.swift [hunk 1]
```diff
@@ -53,7 +53,7 @@ final class SessionUploadService: SessionUpstream {
             } catch {
                 promise(.failure(error))
             }
-            client.requestTask(for: request) { result, request in
+            client.requestTask(for: request) { [self] result, request in
```
Comment (SzymonGorowski) [inline AirCasting/SessionsSynchronization/Adapters/Services/SessionUploadService.swift:56]: Maybe `weak self` there?

File: AirCasting/SessionViews/SessionCartViewModel/DefaultDeleteSessionViewModel.swift [hunk 2]
```diff
@@ -96,7 +96,7 @@ class DefaultDeleteSessionViewModel: DeleteSessionViewModel {
     private func processStreamDeleting() {
         self.measurementStreamStorage.accessStorage { [self] storage in
             guard let sessionToPass = try? storage.getExistingSession(with: session.uuid) else { return }
-            streamRemover.updateSession(session: sessionToPass) { result in
+            streamRemover.updateSession(session: sessionToPass) { [self] result in
```
Comment (SzymonGorowski) [inline AirCasting/SessionViews/SessionCartViewModel/DefaultDeleteSessionViewModel.swift:99]: Maybe `weak self` there?

File: AirCasting/SessionsSynchronization/Adapters/Services/SessionUploadService.swift (hunk 1 above)
Comment (SzymonGorowski) [inline AirCasting/SessionsSynchronization/Adapters/Services/SessionUploadService.swift:56]: As above

**Review by SzymonGorowski — COMMENTED** [review]
Two small changes. Anyway good job

File: AirCasting/Notes/NotesHandler.swift [hunk 3]
```diff
@@ -89,7 +89,7 @@ class NotesHandlerDefault: NSObject, NotesHandler, NSFetchedResultsControllerDel
             do {
                 try storage.updateNote(note, newText: newText, for: sessionUUID)
                 fetchSession(storage: storage) { session in
-                    self.sessionUpdateService.updateSession(session: session) { result in
+                    self.sessionUpdateService.updateSession(session: session) { [self] result in
```
Comment (pawelgil) [inline AirCasting/Notes/NotesHandler.swift:92]: note that this doesn't change anything in practice. The block still captures `self` strongly

File: AirCasting/SessionViews/SessionCartViewModel/DefaultDeleteSessionViewModel.swift (hunk 2 above)
Comment (pawelgil) [inline AirCasting/SessionViews/SessionCartViewModel/DefaultDeleteSessionViewModel.swift:99]: note that this doesn't change anything in practice. The block still captures `self` strongly

File: AirCasting/SessionsSynchronization/Adapters/Services/SessionUploadService.swift (hunk 1 above)
Comment (pawelgil) [inline AirCasting/SessionsSynchronization/Adapters/Services/SessionUploadService.swift:56]: note that this doesn't change anything in practice. The block still captures `self` strongly

**Review by pawelgil — APPROVED** [review]
Idk if those captures should be weak or not, please take a look at the code and decide for yourself

**Pull request merged**